main.go

```diff
@@ -26,6 +27,19 @@ var SlotsInEpoch = uint64(32)
 var UpdateValidatorsIntervalSlots = uint64(600)
 
 func main() {
+	//file is created if not exists, otherwise it appends errors to the existing file
+	//0666 means permisions to read and write to all users, but not execute
+	file, err := os.OpenFile("/oracle-data/oracleLogs.txt", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

Comment:
* lets extract the fileName to a global variable. `LogsFile=oracle_logs.txt` and then use it here
* I would prefer to leave the `oracle-data` folder only for the states. We can have the logs just in /oracle-logs.txt
* when concatenating paths, beter to use something like "filepath.Join(xx, yy)".

Reply:
done! used two global const: logname and logpath